po/sv.po

```diff
@@ -1356,7 +1356,7 @@ msgstr "Symbol"
 #: gnucash/report/trep-engine.scm:925 gnucash/report/trep-engine.scm:1177
 #: gnucash/report/trep-engine.scm:2249
 msgid "Shares"
-msgstr "Andelar"
+msgstr "Aktier"
```

Comment:
@bittin At least this change is going in the wrong direction. Did you consider that the term is also used for funds and other commodities?